### DISCUSSION TITLE
[$20] Finish #685, Antag Names

### DISCUSSION THREAD

**Discussion by ReturnToZender** [Bubberstation/Bubberstation#1885] [ideas/bounties]
Antag names (#685) seeks to make it so Syndicate spawns (like Lone Operative, Lone Infiltrator, Contractor) and Syndicate ghost roles (Dauntless, Interdyne) use a special name pref, a codename applied to the character, to conceal their identity. The former requirement is satisfied, but I can't get the names to show up properly on ghost roles when they spawn. If you can make this happen before I do, I'll pay  you for your work.

**Comment by Odairu** [comment]
This is completed and merged 👍
